packages/x-charts/src/ScatterChart/extremums.ts

```diff
@@ -0,0 +1,64 @@
+import { AxisConfig } from '../models/axis';
+import { ScatterSeriesType } from '../models/seriesType';
+
+type GetExtremumParamsX = {
+  series: { [id: string]: ScatterSeriesType };
+  xAxis: AxisConfig;
+};
+type GetExtremumParamsY = {
+  series: { [id: string]: ScatterSeriesType };
+  yAxis: AxisConfig;
+};
+
+type NUllableExtremum = [number, number] | [null, null];
+type GetExtremumResult = NUllableExtremum;
+
+const mergeMinMax = (acc: NUllableExtremum, val: NUllableExtremum): NUllableExtremum => {
```

Comment:
```suggestion
type NullableExtremum = [number, number] | [null, null];
type GetExtremumResult = NullableExtremum;

const mergeMinMax = (acc: NullableExtremum, val: NullableExtremum): NullableExtremum => {
```
And other places in this file 🙈